### DISCUSSION TITLE
SQL 키워드 정리 노트

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#115] [Announcements]
# 📋 SQL 키워드 정리 노트
- 출제자가 오늘 출제한 문제에서 중요한 키워드를 정합니다
- 해당 키워드에 대한 내용을 링크 or 정리 해서 댓글에 남겨주세요 !

ex 1)
`[[2021.09.07] 문제](sql문제링크)`
`[오늘의 키워드 connect by level](관련내용 링크)`

ex 2)
`[[2021.09.07] 문제](sql문제링크)`
```
오늘의 키워드 정리 내용
```

**Comment by hyk339** [comment]
[2021.09.07] 프로그래머스 [프린터](https://programmers.co.kr/learn/courses/30/lessons/59413?language=oracle)
[오늘의 키워드] [connect by level](https://rocabilly.tistory.com/284)  일렬숫자 생성

**Reply by pyunji** [reply]
[[TO_CHAR 시간 포맷, 날짜 포맷]](https://coding-factory.tistory.com/439)

**Comment by chaeyeon4u** [comment]
[상위 N개 레코드] [ORDER BY 및 ROWNUM 사용 시 유의점](https://ajdahrdl.tistory.com/23)
: 
ROWNUM : 조회된 순서대로 순번을 매기는 고유 변수
ORDER BY : 데이터 정렬
유의점 : ORDER BY로 데이터 정렬이 되기 전에, ROWNUM이 매겨진다.
